lib/hammer_cli/output/output.rb

```diff
@@ -36,6 +36,14 @@ def print_collection(definition, collection)
       adapter.reset_context
     end
 
+    def print_output_type
+      if self.class.adapters.has_key? context[:adapter].to_sym
+        $stderr.puts _("Using '#{context[:adapter]}' adapter")
```

Comment:
I'd prefer method `warn` that we're already [using](https://github.com/theforeman/hammer-cli/blob/master/lib/hammer_cli/utils.rb#L81) instead of using directly `$stderr.puts`. What do you think?